metadata-ingestion/tests/integration/looker/golden_test_ingest_joins.json

```diff
@@ -98,20 +95,18 @@
                                 "auditStamp": {
                                     "time": 0,
                                     "actor": "urn:li:corpuser:unknown",
-                                    "impersonator": null,
-                                    "message": null
+                                    "impersonator": null
                                 },
-                                "dataset": "urn:li:dataset:(urn:li:dataPlatform:looker,lkml_samples.view.my_view_has_no_fields,PROD)",
+                                "dataset": "urn:li:dataset:(urn:li:dataPlatform:looker,lkml_samples.view.my_joined_view_original_name,PROD)",
```

Comment:
looks like UpstreamLineage aspect contents got re-ordered here, that is concerning. Can you check if we are doing a stable sort of the items? 
Common problem observed in the past is using a set()

Reply:
We already do sorting before populating upstreams. ([here](https://github.com/datahub-project/datahub/blob/master/metadata-ingestion/src/datahub/ingestion/source/looker_common.py#L737)) The original golden mces seem to be generated before sorting was in place and somehow(probably, by design) golden tests diff checker gives no diff even if sorting is different. Proposed state of golden files is as generated from update-golden-files flag and as expected according the the sorting code.